From 44b41299557f84602b8a61a1c0e5b34386c16193 Mon Sep 17 00:00:00 2001
From: bogdan <bogdan@bogdan-Studio-XPS-1340>
Date: 05.03.2013 8:39:28



diff --git a/src/jinfodisk/JInfodisk.java b/src/jinfodisk/JInfodisk.java
new file mode 100644
index 0000000..6943ff7
--- /dev/null
+++ b/src/jinfodisk/JInfodisk.java
@@ -0,0 +1,19 @@
+/*
+ * To change this template, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package jinfodisk;
+
+/**
+ *
+ * @author bogdan
+ */
+public class JInfodisk {
+
+    /**
+     * @param args the command line arguments
+     */
+    public static void main(String[] args) {
+        // TODO code application logic here
+    }
+}
